pkg/audit/console_test.go

```diff
@@ -57,7 +58,7 @@ func TestLoggingAuditWrite(t *testing.T) {
 			logger := test.DummyLogger(&output).Sugar()
 
 			audit := &ConsoleAudit{Logger: logger}
-			err := audit.Write(&tc.given)
+			err := audit.Write(context.TODO(), &tc.given)
```

Comment:
i assume the idea here is that console writing hardly ever blocks or has otherwise the need to be canceled, so this noop context is fine?

ah i just saw it is ignore on the called side, so this is merely to satisfy the interface